draft-ietf-ppm-dap.md

```diff
@@ -2139,9 +2139,9 @@ for each input share in the job:
       Collector; the Helper considers a batch to be collected once it has
       responded to an `AggregateShareReq` message from the Leader. A batch is
       determined by query ({{batch-modes}}) conveyed in these messages. Queries
-      must satisfy the criteria covered in {{batch-validation}}. These criteria
-      are meant to restrict queries in a way that makes it easy to determine
-      whether a report pertains to a batch that was collected. See
+      must satisfy the criteria defined by their batch mode. These criteria are
```

Comment:
I think a reference to the batch mode section would be appropriate here.

Reply:
Good idea. I moved the reference from the previous sentence into this one.